test/arm/neon/qrdmulh_n.c

```diff
@@ -212,6 +221,9 @@ test_simde_vqrdmulhq_n_s32 (SIMDE_MUNIT_TEST_ARGS) {
     { { -INT32_C(   219487289),  INT32_C(  1420994589),  INT32_C(   889110344), -INT32_C(  2103115347) },
        INT32_C(  1735639961),
       { -INT32_C(   177394091),  INT32_C(  1148476728),  INT32_C(   718597063), -INT32_C(  1699780598) } },
+    { {              INT32_MIN,              INT32_MIN,              INT32_MIN,              INT32_MIN },
+      {              INT32_MIN,              INT32_MIN,              INT32_MIN,              INT32_MIN },
+      {              INT32_MIN,              INT32_MIN,              INT32_MIN,              INT32_MIN } },
 
```

Comment:
```suggestion
    { {              INT32_MIN,              INT32_MIN,              INT32_MIN,              INT32_MIN },
       INT32_MIN,
      {              INT32_MIN,              INT32_MIN,              INT32_MIN,              INT32_MIN } },

```

Reply:
I have changed the code. But how can I build and run the test cases  in my personal computer ? so I don't have to submit to a CI test every time.

Reply:
> But how can I build and run the test cases in my personal computer ? so I don't have to submit to a CI test every time.

Sorry for the late reply; See https://github.com/simd-everywhere/simde/tree/master/docker#simde-development-container for how to test using multiple architectures on a single system.